pallets/revive/src/address.rs

```diff
@@ -18,61 +18,173 @@
 //! Functions that deal contract addresses.
 
 use alloc::vec::Vec;
```

Comment:
jd: this whole file was copy pasted from a later revive version. The remaining revive pallet fork is outdated.